### DISCUSSION TITLE
Yes / No query in the xmb menu item - uninstall webMAN MOD

### DISCUSSION THREAD

**Discussion by ozapft** [aldostools/webMAN-MOD#647] [Ideas]
It would be very good to have a yes/no query in the menu item - uninstall webMAN MOD. (webMAN_XX.xml)
At the moment a wrong click in the xmb menu is enough and webMAN MOD will be uninstalled. 

A submenu with the two menu entries yes and no would be sufficient
no -> back to the main menu
yes -> uninstall webMAN MOD

**Comment by aldostools** [comment]
Added to commit 8ace3c1425fce73882e828fe2162f3c7bf6a0359
